### ISSUE TITLE
Add option for asking users to confirm big download for webassembly apps

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/panel#3950]
## Request

Add option `--confirmation` to `panel convert` for asking users to confirm big download before starting it.

## Motivation

On the upcoming Panel docs you don't just download pyodide, panel and other dependencies. You ask the user for permission as it is a lot to download.

I believe the same would be useful for apps created via `panel convert`. 

For me it is just *nice to have* right now though.

**Comment by maximlt** [comment]
This sort of flag usually works the other way around, by default you have to confirm and you have to opt in the no confirmation mode (with `--yes` for instance). For the website this made totally sense as you don't want to force people to download that much data (it's a lot more than the average page size), maybe they're in a place where data download is expensive.

This situation is different, to run `panel convert` they have already downloaded `panel` and its dependencies, and the dependencies of the app they're building. At this point I think it's fine to assume they're fine downloading more data. Documenting what happens when you do `panel convert`, and logging the downloads when `panel convert` runs seem like enough information.

**Comment by MarcSkovMadsen** [comment]
Ahh. What you describe is not what I propose.

I propose that when a user of an Panel Webassembly app requests the app they will be asked if they want to continue as it will download a lot of stuff.

Similarly to when you go to the upcoming Panel docs, it does not automatically download pyodide etc. The user has to approve the download.

**Comment by maximlt** [comment]
Ah ok I see, thanks for the details. Seems relevant indeed.
